### PR TITLE
Maven upgrade dependencies

### PR DESCRIPTION
**Pull request by feffi**
Hi, I upgraded some maven dependencies (again) based on maven-versions.

### CODE REVIEW

File: modules/coverage-report/src/test/java/org/jooby/issues/Issue498.java [hunk 1]
```diff
@@ -6,7 +6,9 @@
 import org.jooby.test.ServerFeature;
 import org.junit.Test;
 
-import com.couchbase.client.deps.io.netty.util.internal.chmv8.ForkJoinPool;
+import java.util.concurrent.ForkJoinPool;
+
+//import com.couchbase.client.deps.io.netty.util.internal.chmv8.ForkJoinPool;
```
Comment (jknack) [inline modules/coverage-report/src/test/java/org/jooby/issues/Issue498.java]: remove the commented line

File: modules/coverage-report/src/test/java/org/jooby/issues/Issue616.java [hunk 2]
```diff
@@ -9,8 +9,8 @@
   public static class Person {
 
     // mandatory fields
-    private final String name;
-    private final String surname;
+    private String name = null;
```
Comment (jknack) [inline modules/coverage-report/src/test/java/org/jooby/issues/Issue616.java]: why these changes?

File: modules/jooby-couchbase/src/main/java/org/jooby/couchbase/Couchbase.java [hunk 3]
```diff
@@ -203,7 +203,8 @@
  */
 package org.jooby.couchbase;
 
-import com.couchbase.client.deps.io.netty.util.internal.MessagePassingQueue.Supplier;
+
+import com.couchbase.client.deps.io.netty.util.internal.shaded.org.jctools.queues.MessagePassingQueue;
```
Comment (jknack) [inline modules/jooby-couchbase/src/main/java/org/jooby/couchbase/Couchbase.java]: Usage of `com.couchbase.client.deps.io.netty.util.internal.MessagePassingQueue.Supplier` is wrong, should be a `java.util.function.Supplier`

File: modules/coverage-report/src/test/java/org/jooby/issues/Issue498.java (hunk 1 above)
Comment (feffi) [reply modules/coverage-report/src/test/java/org/jooby/issues/Issue498.java]: Sure, it’s just the output of versions plugin basically. Just leave thin on hold, until i‘m done. I will compile a release note then. About 20-30 more to come, but a full testing cycle needs some time ;-)

File: modules/jooby-couchbase/src/main/java/org/jooby/couchbase/Couchbase.java (hunk 3 above)
Comment (feffi) [reply modules/jooby-couchbase/src/main/java/org/jooby/couchbase/Couchbase.java]: sure, fixed in https://github.com/jooby-project/jooby/pull/1184/commits/60487b36bd90be4e16a3a1d67030891c166074c0

File: modules/coverage-report/src/test/java/org/jooby/issues/Issue616.java (hunk 2 above)
Comment (feffi) [reply modules/coverage-report/src/test/java/org/jooby/issues/Issue616.java]: This was because of the Mock init needs a default constructor, else it won't init the corresponding test. But I could be wrong.

**Comment by feffi** [comment]
Hi Edgar,

I'm done for now. You could close and merge, if you want. More to come, but for the sake of api changes, the rest of the updates will come in separate PRs.

**Comment by feffi** [comment]
ok, really done now :-) found the (real) breaking change in mongodb-rx-client

**Comment by jknack** [comment]
Cool, will looks later, but we will need to wait for 1.5.0 release (next will be 1.4.2)

**Comment by jknack** [comment]
Still need a dump of what has been updated: `libname -> current version -> new version`

and remember it is easy to merge (and we don't need a dump) if we send a pull for each library upgrade... I now sometimes isn't need it but it helps when writing release notes and/or to find issues after upgrade.

Also, would you like to do code reviews? I can start sending pull request again.

Thanks

**Comment by feffi** [comment]
Hi,
sure, if you like us to review your stuff also, I would be pleased ;-) Next time, I'll splitt this updates into single ones. But sometime several libs have to be pulled up occordingly, so splitting by single lib won't work here. I'll try to keep it as small as possible.

**Comment by feffi** [comment]
Here you go, a compiled list of all dependency upgrades in this PR:
```
cglib:cglib-nodep ............................................. 3.2.0 -> 3.2.6
ch.qos.logback:logback-classic ................................ 1.2.3 -> 1.3.0-alpha4
com.amazonaws:aws-java-sdk-core ............................... 1.11.273 -> 1.11.355
com.amazonaws:aws-java-sdk-s3 ................................. 1.11.273 -> 1.11.355
com.couchbase.client:java-client .............................. 2.3.1 -> 2.5.9
com.datastax.cassandra:cassandra-driver-core .................. 3.4.0 -> 3.5.0
com.datastax.cassandra:cassandra-driver-extras ................ 3.4.0 -> 3.5.0
com.datastax.cassandra:cassandra-driver-mapping ............... 3.4.0 -> 3.5.0
com.fasterxml.jackson.core:jackson-core ....................... 2.9.5 -> 2.9.6
com.fasterxml.jackson.core:jackson-databind ................... 2.9.5 -> 2.9.6
com.fasterxml.jackson.dataformat:jackson-dataformat-yaml ...... 2.9.5 -> 2.9.6
com.fasterxml.jackson.datatype:jackson-datatype-guava ......... 2.9.5 -> 2.9.6
com.fasterxml.jackson.datatype:jackson-datatype-jdk8 .......... 2.9.5 -> 2.9.6
com.fasterxml.jackson.datatype:jackson-datatype-jsr310 ........ 2.9.5 -> 2.9.6
com.fasterxml.jackson.module:jackson-module-afterburner ....... 2.9.5 -> 2.9.6
com.fasterxml.jackson.module:jackson-module-kotlin ............ 2.9.5 -> 2.9.6
com.fasterxml.jackson.module:jackson-module-parameter-names ... 2.9.5 -> 2.9.6
com.fizzed:rocker-compiler .................................... 0.23.0 -> 0.24.0
com.fizzed:rocker-runtime ..................................... 0.23.0 -> 0.24.0
com.github.ben-manes.caffeine:caffeine ........................ 2.6.1 -> 2.6.2
com.github.davidmoten:rxjava-jdbc ............................. 0.7 -> 0.7.16
com.github.jknack:handlebars .................................. 4.0.7 -> 4.1.0
com.github.jknack:handlebars-guava-cache ...................... 4.0.7 -> 4.1.0
com.github.lalyos:jfiglet ..................................... 0.0.7 -> 0.0.8
com.github.sommeri:less4j ..................................... 1.13.0 -> 1.17.2
com.google.code.gson:gson ..................................... 2.8.2 -> 2.8.5
com.google.javascript:closure-compiler ........................ v20180204 -> v20180610
com.graphaware.neo4j:expire ................................... 3.3.2.52.4 -> 3.4.0.52.4
com.graphaware.neo4j:runtime .................................. 3.3.2.52 -> 3.4.0.52
com.h2database:h2 ............................................. 1.4.196 -> 1.4.197
com.hazelcast:hazelcast ....................................... 3.9.2 -> 3.10.2
com.ning:async-http-client .................................... 1.9.36 -> 1.9.40
com.orbitz.consul:consul-client ............................... 1.0.1 -> 1.2.1
com.typesafe:config ........................................... 1.3.2 -> 1.3.3
com.typesafe.akka:akka-actor_2.12 ............................. 2.5.9 -> 2.5.13
com.typesafe.akka:akka-slf4j_2.12 ............................. 2.5.9 -> 2.5.13
cz.jiripinkas:jsitemapgenerator ............................... 2.1 -> 2.2
de.undercouch:bson4jackson .................................... 2.9.0 -> 2.9.2
io.ebean:ebean ................................................ 11.16.1 -> 11.18.1
io.github.lukehutch:fast-classpath-scanner .................... 2.8.1 -> 3.1.5
io.reactivex:rxjava-string .................................... 1.1.0 -> 1.1.1
io.requery:requery ............................................ 1.4.0 -> 1.5.1
io.requery:requery-kotlin ..................................... 1.4.0 -> 1.5.1
net.sf.ehcache:ehcache ........................................ 2.10.0 -> 2.10.5
net.spy:spymemcached .......................................... 2.12.0 -> 2.12.3
org.apache.commons:commons-lang3 .............................. 3.4 -> 3.7
org.apache.httpcomponents:fluent-hc ........................... 4.5.2 -> 4.5.5
org.apache.httpcomponents:httpclient .......................... 4.5.2 -> 4.5.5
org.apache.httpcomponents:httpmime ............................ 4.5.2 -> 4.5.5
org.apache.maven:maven-core ................................... 3.3.3 -> 3.5.4
org.apache.maven:maven-plugin-api ............................. 3.3.3 -> 3.5.4
org.crashub:crash.connectors.ssh .............................. 1.3.1 -> 1.3.2
org.crashub:crash.connectors.telnet ........................... 1.3.1 -> 1.3.2
org.crashub:crash.connectors.web .............................. 1.3.1 -> 1.3.2
org.crashub:crash.plugins.cron ................................ 1.3.1 -> 1.3.2
org.crashub:crash.plugins.mail ................................ 1.3.1 -> 1.3.2
org.crashub:crash.shell ....................................... 1.3.1 -> 1.3.2
org.jooq:jooq ................................................. 3.7.3 -> 3.11.2
org.jooq:jooq-codegen ......................................... 3.7.3 -> 3.11.2
org.jsoup:jsoup ............................................... 1.8.3 -> 1.11.3
org.mongodb.morphia:morphia ................................... 1.1.1 -> 1.3.2
org.mongodb.morphia:morphia-logging-slf4j ..................... 1.1.1 -> 1.3.2
org.quartz-scheduler:quartz ................................... 2.2.2 -> 2.3.0 
org.twdata.maven:mojo-executor ................................ 2.2.0 -> 2.3.0
org.zeroturnaround:zt-zip ..................................... 1.9 -> 1.13
io.ebean:ebean ................................................ 11.18.1 -> 11.18.2
com.amazonaws:aws-java-sdk-core ............................... 1.11.355 -> 1.11.356
com.amazonaws:aws-java-sdk-s3 ................................. 1.11.355 -> 1.11.356
org.flywaydb:flyway-core ...................................... 5.0.7 -> 5.1.3
org.apache.maven:maven-plugin-api ............................. 3.3.3 -> 3.5.4
org.apache.maven.plugin-tools:maven-plugin-annotations ........ 3.4 -> 3.5.2
io.dropwizard.metrics:metrics-core ............................ 3.1.2 -> 4.1.0-rc2, fixed #1183
io.dropwizard.metrics:metrics-healthchecks .................... 3.1.2 -> 4.1.0-rc2, fixed #1183
io.dropwizard.metrics:metrics-jvm ............................. 3.1.2 -> 4.1.0-rc2, fixed #1183
org.jooby:jooby-jdbi3 ......................................... 3.1.2 -> 3.3.0
org.jruby:jruby-complete ...................................... 9.0.1.0 -> 9.2.0.0
org.jetbrains.kotlin:kotlin-stdlib ............................ 1.2.41 -> 1.2.50
org.jetbrains.kotlin:kotlin-stdlib-jdk7 ....................... 1.2.41 -> 1.2.50
org.jetbrains.kotlin:kotlin-stdlib-jdk8 ....................... 1.2.41 -> 1.2.50
org.jetbrains.kotlin:kotlin-reflect ........................... 1.2.41 -> 1.2.50
io.github.lukehutch:fast-classpath-scanner .................... 3.1.5 -> 3.1.6
org.codehaus.mojo:exec-maven-plugin ........................... 1.3.2 -> 1.6.0
org.webjars:swagger-ui ........................................ 3.14.2 -> 3.17.1
org.unbescape:unbescape ....................................... 1.1.3.RELEASE -> 1.1.6.RELEASE
org.mongodb:mongo-java-driver ................................. 3.2.2 -> 3.8.0
org.mongodb:mongodb-driver-async .............................. 3.2.2 -> 3.8.0
org.mongodb:mongodb-driver-core ............................... 3.2.2 -> 3.8.0
```
